Makefile

```diff
@@ -224,6 +224,7 @@ format: go-fmt jsonnet-fmt check-license shellcheck
 .PHONY: go-fmt
 go-fmt:
 	go fmt $(pkgs)
+	gofmt -s -w .
```

Comment:
do we need to run both?

Reply:
I think we can switch to `gofmt` only, but I am not 100% sure.

Reply:
SO to the rescue :) https://stackoverflow.com/a/55319554
Lets remove `go fmt`

Reply:
sgtm! updated to remove `go fmt`